packages/fluentui/docs/src/examples/components/Breadcrumb/Usage/index.tsx

```diff
@@ -0,0 +1,15 @@
+import * as React from 'react';
+import ComponentExample from '../../../../components/ComponentDoc/ComponentExample';
+import ExampleSection from '../../../../components/ComponentDoc/ExampleSection';
+
+const Usage = () => (
+  <ExampleSection title="Usage">
```

Comment:
This should be under `Types` section. You take check `Slider` docs for inspiration https://fluentsite.z22.web.core.windows.net/components/slider/definition

Reply:
Moved by commit 
[feat(Breadcrumb): default example + icon divider example](https://github.com/microsoft/fluentui/pull/14926/commits/7fd8750359ab838b30235fc4ab2bc2b3eb35beb6)